src/core/include/antioch/chemical_mixture.h

```diff
@@ -569,6 +585,24 @@ namespace Antioch
     return;
   }
 
+  template<typename CoeffType>
+  inline
+  const ChemicalMixture<CoeffType> & ChemicalMixture<CoeffType>::chemical_mixture() const
+  {
+     antioch_deprecated();
+     return *this;
+  }
+
+  template<typename CoeffType>
+  template<typename ThermoEvaluator>
+  inline
+  const ThermoEvaluator & ChemicalMixture<CoeffType>::thermo_evaluator() const
+  {
+     std::cerr << "What the hell are you doing here? This backward compatibility place should not be explored" << std::endl;
```

Comment:
Not a nice message.  Output deep in the library needs to be wrapped in antioch_do_once() (as soon as we port over libmesh_do_once) so it can't end up being spammy.

In this case, though, the output should just be "antioch_deprecated();"